tests/test_04_dxf_high_level_structs/test_414_explode_blockrefs.py

```diff
@@ -155,5 +155,23 @@ def test_05_examine_uniform_scaled_ellipse(doc, msp):
     assert ellipse.dxf.ratio == 0.5
 
 
+def test_06_return_skipped_entities(doc, msp):
```

Comment:
As commented in explode.py, ATTDEF is special and does not really belong to the block content. It is just a template to create attached ATTRIB entities.